Installing-and-Configuring/OS/Generic.md

```diff
@@ -0,0 +1,118 @@
+---
+layout: default
+title: Generic
+permalink: /OS/Generic/
+parent: Operating Systems
+grand_parent: Installing and configuring
+nav_order: 1
+---
+
+<!-- markdownlint-disable MD033 -->
+<details open markdown="block">
+  <summary>
+    Table of contents
+  </summary>
+  {: .text-delta }
+1. TOC
+{:toc}
+</details>
+<!-- markdownlint-enable MD033 -->
+
+Generic OS Installation
+===
+
+Insert OS installation media into one of the USB3 ports (blue on Thinkpads).
+ Turn the system on and navigate to the USB boot in the graphical heads menu. 
+
+Main Menu => Boot Options => USB Boot
+
+From here you will be presented with a menu of the boot options on the USB media.  Choose the one you wish to use and Press enter.  It is your responsibility to verify the integrity of the media before using it.  Check your downloads against the signatures provided by vendors.
+
+
+Securely Booting Installation Media
+----
+
+Heads also supports booting directly from verified ISOs on a standard disk partition.  In this scenario heads will check the ISO against a corresponding signature.  The ISO must be signed by a valid key for the boot process to succeed.  
```

Comment:
Heads also supports booting directly from verified ISOs on a standard disk partition.  In this scenario heads will check the ISO against a detached PGP signature file.  The ISO must be signed by a valid key for the boot process to succeed, let it be the distribution signing key or your own.

Reply:
incorporated.  push in a few minutes.